### PR TITLE
144187913 Implement Filter and Sort of Search Results

### PR DESCRIPTION
**Pull request by taiwo-sam**
This PR implements the sorting and filtering of search results. When a user searches for a product, they can further refine the search results by sorting and filtering using such criteria as price, vendor, tags etc.

### CODE REVIEW

File: .meteor/packages [hunk 1]
```diff
@@ -16,7 +16,7 @@ audit-argument-checks@1.0.7   # ensure meteor method argument validation
 browser-policy@1.1.0          # security-related policies enforced by newer browsers
 juliancwirko:postcss          # CSS post-processing plugin (replaces standard-minifier-css)
 abernix:standard-minifier-js  # a minifier plugin used for Meteor apps by default
-session@1.1.7                 # ReactiveDict whose contents are preserved across Hot Code Push
+session                 # ReactiveDict whose contents are preserved across Hot Code Push
```
Comment (ghost) [inline .meteor/packages:19]: Any reason you removed the version?

File: imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js [hunk 2]
```diff
@@ -40,19 +41,90 @@ Template.searchModal.onCreated(function () {
...
-        const productResults = ProductSearch.find().fetch();
+        let productResults = ProductSearch.find().fetch();
+
+        // Only display sort and filter options if there are search results and a search query
+        if (productResults.length > 0 && searchQuery.length > 0) {
+          Session.set("displaySortandFilter", true);
+        } else {
+          Session.set("displaySortandFilter", false);
+        }
+        // }
```
Comment (ghost) [inline imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js]: Can you please remove the commented brace?

File: .meteor/packages (hunk 1 above)
Comment (taiwo-sam) [reply .meteor/packages:19]: None. I installed session anew. I didn't know it was already installed.

File: imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js (hunk 2 above)
Comment (taiwo-sam) [reply imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js]: Done. @andela-oisola

**Review by ghost — APPROVED** [review]

**Pull request merged**